### PR TITLE
PPD-21 - add production namespace for prod pin phone monitoring service

### PR DESCRIPTION
**Pull request by garethmdavies**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/iam.tf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+data "aws_iam_policy_document" "bt_upload_policy" {
+  statement {
+    actions = ["s3:PutObject"]
+
+    resources = [
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.mp3",
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.flac",
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.wav",
+      "${module.hmpps_pin_phone_monitor_document_s3_bucket.bucket_arn}/*.xml",
```
Comment (brightonsbox) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/iam.tf:18]: Maybe get rid of xml? We wouldn't be able to process any xml files so stop them sending entirely?

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/s3.tf [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  }
+
+  lifecycle_rule = [
+    {
+      enabled = true
+      id      = "pin-phone-recording-expiry"
+      prefix  = "recordings/"
+      expiration = [
+        {
+          days = 90
```
Comment (brightonsbox) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/s3.tf:24]: 👍

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/s3.tf [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  }
+
+  queue {
+    id        = "recording-deletion-event"
+    queue_arn = module.hmpps_pin_phone_monitor_s3_event_queue.sqs_arn
+    events = [
+    "s3:ObjectRemoved:Delete"]
+    filter_prefix = "recordings/"
+    filter_suffix = ".flac"
+  }
```
Comment (brightonsbox) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-pin-phone-monitor-prod/resources/s3.tf:141]: I don't think we're doing anything with this event in code yet, should we be?

**Review by brightonsbox — DISMISSED** [review]
LGTM, a couple of small comments / questions

**Review by razvan-moj — APPROVED** [review]

**Pull request merged**